pyNorm/io/pyn_io.py

```diff
@@ -1,3 +1,192 @@
+def save_pyn_results(spec,filename = 'None'):
```

Comment:
I wonder if this could be simply save_pyn.

Reply:
I'm going to reverse myself and recommend `pyn_save`. I'll make the change after merging / committing, but be aware.